poi_fr.md

```diff
@@ -0,0 +1,72 @@
+Navitia POI version 0.1
+=======================
+
+# Introduction
+
+Les POI permettent d'enrichir Navitia avec des points de départ ou d'arrivée
+connus des usagers pour la rubrique "Itinéraires" ou la présentation des lieux
+publics ou des services à proximités d'un lieu.
+
+Cette spécification décrit le format d'échange de ces POI entre Fusio et
+Navitia2, et le format d'alimentation cible du nouveau NMM. Certains POI ont
```

Comment:
Fusio is no longer the only supplier of POI, and I’m not sure NMM still exists

Reply:
I can just change it for something like

```suggestion
Cette spécification décrit le format d'échange de ces POI vers
Navitia2. Certains POI ont
```

Reply:
That’s exactly what I was thinking.

Reply:
Done in 590207cda6692b3e3e93d8340354bb6021802cf1.